documentation/faq/10-what-can-i-create.md

```diff
@@ -0,0 +1,19 @@
+---
+title: What kinds of applications can I make with SvelteKit?
+---
+
+SvelteKit can be used to create several kinds of application with the support of [third-party tools](#integrations). Out of the box, SvelteKit supports many features including:
```

Comment:
```suggestion
SvelteKit can be used to create most kinds of applications with the support of [third-party tools](#integrations). Out of the box, SvelteKit supports many features including:
```

I am not the biggest fan of saying third-party tools but I think it gets the message across - maybe we can instead mention integrations after the bullet points?